test/resources/unnormalize.css

```diff
@@ -1,14 +1,18 @@
 body {
     font-size: 24px;
     font-family: Courier, monospace;
```

Comment:
comic sans might be better here. (this is a real comment)

Reply:
Updated.